pkg/provision/local_store.go

```diff
@@ -109,7 +109,13 @@ func NewFSStore(root string) (*FSStore, error) {
 }
 
 func (s *FSStore) removeAllButPersistent(rootPath string) error {
-	err := filepath.Walk(rootPath, func(path string, info os.FileInfo, r error) error {
+	// if rootPath is not present on the filesystem, return
+	_, err := os.Stat(rootPath)
```

Comment:
This fix is not very useful, since you still return an error if the rootPath does not exist.
A correct solution would be 

```suggestion
	_, err := os.Stat(rootPath)
	if os.IsNotExist(err) {
	   return nil
	} else if err != nil {
	   return err
	}
```